README.md

```diff
@@ -208,6 +209,14 @@ Each package places lldap's configuration file at `/etc/lldap.toml` and offers
 [systemd service](https://wiki.archlinux.org/title/systemd#Using_units)
 `lldap.service` to (auto-)start and stop lldap.
 
+### With FreeBSD
+
+You can also install it as a rc.d service in FreeBSD, see
+[FreeBSD-install.md](example_configs/freebsd/freebsd-install.md).
+
+The rc.d script file 
+[rcd_lldap](example_configs/freebsd/rcd_lldap).
```

Comment:
Link should be to `rc.d_lldap`, not `rcd_lldap`.

You can check the links by clicking the three dots ... in the upper right corner, then View the file: you'll get the rendered markdown, and you can click on the links (which, being relative, will find the files in your PR)

Reply:
Yepp, mistyped, beside looking previews. Thanks! Fixed.